packages/flutter_tools/lib/src/linux/linux_workflow.dart

```diff
@@ -21,10 +21,10 @@ class LinuxWorkflow implements Workflow {
   bool get appliesToHostPlatform => platform.isLinux;
```

Comment:
Shouldn't these be gated on it being enabled too? That's how web is done, and it seems odd to be inconsistent about it. That would also simplify other code, since, e.g., the doctor conditions wouldn't need to check the feature flag.

Reply:
Yeah, moving the check here simplifies the rest of the code